pom.xml

```diff
@@ -41,7 +41,9 @@
     <bahmniappsRevision>b954149</bahmniappsRevision>
 
     <!-- In dev -->
-    <initializerVersion>2.1.0</initializerVersion>
+    <initializerVersion>2.1.0-SNAPSHOT</initializerVersion>
```

Comment:
Please make sure to pull latest changes before making the PR, so let's keep `2.1.0` here